solution/backend/resources/v3views.py

```diff
@@ -231,6 +231,8 @@ class ResourceExplorerViewSetMixin(OptionalPaginationMixin, LocationFiltererMixi
                               "search with quotes.", str, False),
         OpenApiQueryParameter("categories", "Limit results to only resources found within these categories. Use "
                               "\"&categories=X&categories=Y\" for multiple.", int, False),
+        OpenApiQueryParameter("sort", "Sort results by this field. Valid values are \"newest\", \"oldest\", and \"relevance\". "
```

Comment:
Seeing "newest" and "oldest" in a couple of places, but the queryset only has `if X sort by relevance, else sort by date_annotated`

Reply:
reference to "oldest" has been removed until it is implemented in a future story.